src/components/markdown/markdown-parser.ts

```diff
@@ -35,7 +35,14 @@ export async function markdownToHTML(
         .use(remarkRehype, { allowDangerousHtml: true })
         .use(rehypeExternalLinks as any, { target: '_blank' })
         .use(rehypeRaw)
-        .use(rehypeSanitize)
+        .use(rehypeSanitize as any, {
+            attributes: {
+                '*': ['style'],
+            },
+            style: {
+                color: /^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$/,
```

Comment:
It seems like this isn't filtering anything 🙁

<img width="570" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/6d2d9a7e-9742-42dd-b913-45c9288f157f">

I'll look further into this tomorrow.

Reply:
Ahh, I didn't test it further. I blame chatgpt for the syntax 😄

Reply:
⚡